### ISSUE TITLE
time out on folder sync, due to corrupt content-type header

### ISSUE THREAD

**Issue by msftrncs** [thunderbird/thunderbird-android#2336]
Folder synchronization should complete, not time out. (was previously working fine syncing > 400 messages)

### Steps to reproduce
1. synchronize a folder with a large message (>2mb) from SmarterMail.
2. wait for time-out

### Environment
K-9 Mail version: 5.203

Android version: 6.0.1

Account type: IMAP on SmarterMail 13.0

K9 displays socket exception time out on my inbox when synchronization is set to 3 weeks back.  Folder sync is set to 64K.  A message approximately 3 weeks back that K9 hasn't sync yet includes a large 2.5mb PDF attachment. Not 100% sure but this message is an oddity as practically all other messages are under 256kb.

**Comment by msftrncs** [comment]
Actually was able to determine the culprit email msg was a 128k message from CNET.  Moved out of the inbox and the sync completes, moved back in and it fails again.  No other message from CNET is causing an issue.   The message displays fine in SmarterMail webmail, but looking at the message raw, it seems there is corruption in the headers ...

The following line went on for about 2500 bytes of HTML code, so I am sure this confused K9's header processing. (I doubled the slashes to disable the HTML codes).   
----
Content-Type: text/html; charset="POP CULTURE, DELIVERED /<//td>               <//tr> 
...
<//div>"
----
